Sources/StreamChat/Database/DTOs/MessageDTO.swift

```diff
@@ -68,6 +69,11 @@ class MessageDTO: NSManagedObject {
     override func willSave() {
         super.willSave()
         
+        if let channel = previewIn, !channel.hasChanges, isUpdated {
+            let cid = channel.cid
+            channel.cid = cid
+        }
```

Comment:
What is this actually doing? 🤔 Can we provide a comment?

Reply:
When the preview message is updated (e.g. edited) the channel cell should be reloaded to reflect the latest preview state. But the update of the source entity (message) does not trigger an update on the destination entity (channel) in 1-1 relation so we manually mark the channel as `dirty`. I'll leave a comment 👍